test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Builder/Conventions/ODataConventionModelBuilderTests.cs

```diff
@@ -3013,6 +3013,58 @@ public void GetEdmModel_WorksOnConventionModelBuilder_ForDerivedOpenEntityType()
             Assert.Equal(baseDynamicPropertyAnnotation.PropertyInfo.Name, derivedDynamicPropertyAnnotation.PropertyInfo.Name);
         }
 
+        [Fact]
+        public void GetEdmModel_WorksOnConventionModelBuilder_ForEntityType_InstanceAnnotation()
+        {
+            // Arrange
+            ODataConventionModelBuilder builder = ODataConventionModelBuilderFactory.Create();
+            builder.EntityType<EntityTypeWithAnnotation>();
+
+            // Act
+            IEdmModel model = builder.GetEdmModel();
+
+            // Assert
+            Assert.NotNull(model);
+            IEdmEntityType entityType =
+                Assert.Single(model.SchemaElements.OfType<IEdmEntityType>().Where(c => c.Name == "EntityTypeWithAnnotation"));
+            Assert.Single(entityType.Properties());
+
+            InstanceAnnotationContainerAnnotation instanceAnnoteDict =
+                model.GetAnnotationValue<InstanceAnnotationContainerAnnotation>(entityType);
+
+            Assert.Equal("InstanceAnnotations", instanceAnnoteDict.PropertyInfo.Name);
+        }
+
+
+        [Fact]
+        public void GetEdmModel_WorksOnConventionModelBuilder_ForDerivedEntityType()
+        {
+            // Arrange
+            ODataConventionModelBuilder builder = ODataConventionModelBuilderFactory.Create();
+            builder.EntityType<EntityTypeWithAnnotation>();
+
+            // Act
+            IEdmModel model = builder.GetEdmModel();
+
+            // Assert
+            Assert.NotNull(model);
+            IEdmEntityType baseEntityType =
+                Assert.Single(model.SchemaElements.OfType<IEdmEntityType>().Where(c => c.Name == "EntityTypeWithAnnotation"));
+            Assert.Single(baseEntityType.Properties());
+
+            IEdmEntityType derivedEntityType =
+                Assert.Single(model.SchemaElements.OfType<IEdmEntityType>().Where(c => c.Name == "DerivedEntityTypeWithAnnotation"));
+            Assert.Equal(2, derivedEntityType.Properties().Count());
+
+            InstanceAnnotationContainerAnnotation basePropertyAnnotation =
+                model.GetAnnotationValue<InstanceAnnotationContainerAnnotation>(baseEntityType);
+
+            InstanceAnnotationContainerAnnotation derivedPropertyAnnotation =
+                model.GetAnnotationValue<InstanceAnnotationContainerAnnotation>(derivedEntityType);
```

Comment:
Why can we get the container annotation from the derived type? 

It doesn't make sense.  @mikepizzo, what' s your thought?